fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -708,6 +710,14 @@ private void handleQueryStmt(ExecPlan execPlan) throws Exception {
         TDescriptorTable descTable = execPlan.getDescTbl().toThrift();
         List<String> colNames = execPlan.getColNames();
         List<Expr> outputExprs = execPlan.getOutputExprs();
+        int colId = 0;
+        for (Expr expr : outputExprs) {
+            if (expr.getType().getPrimitiveType().equals(PrimitiveType.UNKNOWN_TYPE)) {
+                throw new SemanticException("Column " + colNames.get(colId) +
+                        " convert failed, and column type is known!");
+            }
+            colId++;
+        }
```

Comment:
You needn't do this check here, this case will be checked when the sql is analyzed. We should make sure that  all unknown type will be checked when analyzing.

Reply:
> You needn't do this check here, this case will be checked when the sql is analyzed. We should make sure that all unknown type will be checked when analyzing.

Got.Now I have removed these, and check SQL contains star like `select * from binarytest` checks to `SelectAnalyzer.analyzeSelect()`